tasks/managed/embargo-check/embargo-check.yaml

```diff
@@ -99,14 +99,31 @@ spec:
             fi
             set -x
 
-            # Perform additional checks only for Vulnerability issues for issues.redhat.com
+            # Perform additional checks only for issues for issues.redhat.com
             if [ "$server" != "issues.redhat.com" ] ; then
                 continue
             fi
+
+            # public should be true if and only if the security field doesn't exist and the issue is accessible
```

Comment:
Nit: Capital P in Public for consistency.

Reply:
fixed thanks